test/models/compute_resources/ovirt_test.rb

```diff
@@ -13,6 +13,18 @@ class OvirtTest < ActiveSupport::TestCase
     assert_equal host, as_admin { cr.associated_host(vm) }
   end
 
+  describe 'validate' do
+    test 'valid if url is present' do
```

Comment:
Okay, I have added `url_schema` for VMware as well. And have asked on IRC for some help to know about other CRs.